packages/react-components/react-tree/stories/Tree/FlatTree.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import {
-  FlatTree as Tree,
+  FlatTree,
```

Comment:
These imports should be modified to come from react-components since the package is stable. When the imports don't come from react-components that's when the renaming happens I think